.github/workflows/storybook-chromatic.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         runs-on: ubuntu-latest
         timeout-minutes: 30
         container:
-            image: mcr.microsoft.com/playwright:v1.29.2 # Same as @storybook/test-runner@0.13's dependency
+            image: mcr.microsoft.com/playwright:v1.32.2 # Same as @storybook/test-runner@0.15.2's dependency
```

Comment:
I only bumped to `1.32.2` based on what the `@storybook/test-runner` [had in the latest version](https://github.com/storybookjs/test-runner/blob/next/yarn.lock#L11313) (v0.15.2)